src/mam/mod_mam_riak_timed_arch_yz.erl

```diff
@@ -122,20 +122,26 @@ stop_muc_archive(Host) ->
     ejabberd_hooks:delete(mam_muc_remove_archive, Host, ?MODULE, remove_archive, 50),
     ok.
 
+%% LocJID - From/To JID depending on direction of message (single message is saved in two directions)
```

Comment:
If we're going to explain these here, I think it's more accurate to say:
* `LocJID` - archive owner's JID
* `RemJID` - interlocutor's JID